torchvision/models/detection/retinanet.py

```diff
@@ -386,7 +385,7 @@ def compute_loss(self, targets, head_outputs, anchors):
         matched_idxs = []
         for anchors_per_image, targets_per_image in zip(anchors, targets):
             if targets_per_image['boxes'].numel() == 0:
-                matched_idxs.append(torch.empty((0,)))
+                matched_idxs.append(torch.empty((0,), dtype=torch.int32))
```

Comment:
I believe this should be `int64`?

Reply:
> I believe this should be `int64`?

Why do you think that?

Reply:
Because indices are normally `int64` in PyTorch

Reply:
Yep I think you're correct. `matched_idxs` comes from `Matcher`, which indeed returns `int64` (https://github.com/pytorch/vision/blob/master/torchvision/models/detection/_utils.py#L272). Do you want me to make the change or were you handling these last bits?

Reply:
I'm handling those last bits, no worries :-)